src/main/java/com/openlattice/search/SearchService.java

```diff
@@ -226,6 +228,7 @@ public void deleteEntities( EntitiesDeletedEvent event ) {
     @Timed
     @Subscribe
     public void deleteLinkedEntities( LinkedEntitiesDeletedEvent event ) {
+        // TODO figure how we handle deletion of entities which belong to a linking entity
```

Comment:
do we have a ticket for this?

Reply:
Yes! https://jira.openlattice.com/browse/LATTICE-2225
This ticket will handle both normal and linked entity deletions in elasticsearch